Login-01/angular.json

```diff
@@ -3,7 +3,7 @@
   "version": 1,
   "newProjectRoot": "projects",
   "projects": {
-    "example-angular": {
+    "angular-blog": {
```

Comment:
this doesnt need to be named after the blog. can just be 'login-example' or whatever